<?xml version="1.0"?>
<launch>

  <!-- <arg name="base_global_planner" default="navfn/NavfnROS"/> -->
  <arg name="base_global_planner" default="rrt_star_global_planner/RRTStarPlanner"/>
  <!-- <arg name="base_local_planner" default="base_local_planner/TrajectoryPlannerROS"/> -->
  <arg name="base_local_planner" default="dwa_local_planner/DWAPlannerROS"/>
  
  
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

    <param name="base_global_planner" value="$(arg base_global_planner)"/>
    <param name="base_local_planner" value="$(arg base_local_planner)"/>  
    
    <rosparam file="$(find rrt_star_global_planner)/config/common_costmap.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find rrt_star_global_planner)/config/common_costmap.yaml" command="load" ns="local_costmap" />
    
    <rosparam file="$(find rrt_star_global_planner)/config/local_costmap.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find rrt_star_global_planner)/config/global_costmap.yaml" command="load" ns="global_costmap" /> 

    <rosparam file="$(find rrt_star_global_planner)/config/planner.yaml" command="load"/>

  </node>

</launch>